index.html

```diff
@@ -163,6 +163,38 @@ <h3>Production packages</h3>
     <h2>Who we work with</h2>
     <a name="Customers"></a>
   </div>
+  <div class="bg-accent-warm-light padding-4">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-2"></div>
+      <div class="tablet:grid-col-2">
+        <img src="{{ site.baseurl }}/assets/images/logos/EPA-logo.svg" alt="EPA logo" class="square-10"/>
+      </div>
+      <div class="tablet:grid-col-8">
+        <h3>Environmental Protection Agency - AirNow.gov</h3>
+        <h4>EPA AirNow Application</h4>
+      </div>
+      <div class="tablet:grid-col-2"></div>
+    </div>
+    <div class="grid-row usa-prose">
+      <div class="tablet:grid-col-2"></div>
+        <div class="tablet:grid-col border-top border-base-light">
+        <p>
+          During the Canadian wildfires in the summer of 2023 occurred, Phil Dickerson, Leader at the Information Transfer Group - Office of Air Quality Planning and Standards which operates EPA AirNow reported that their customer traffic “… surged faster than we’ve ever seen, it probably went up by a million hits in one day.” He continued: “On a Tuesday morning everything looked normal and by Wednesday afternoon we were at a new record high for traffic, which was then broken on Thursday, it happened that fast”. The EPA AirNow team attributed the stability of the AirNow application <a href="https://www.airnow.gov">airnow.gov</a> to their ability to easily scale their applications on Cloud.gov. 
```

Comment:
```suggestion
          During the Canadian wildfires in the summer of 2023, traffic to <a href="https://www.airnow.gov">AirNow.gov</a> surged to several orders of magnitude higher than ever before. But thanks to the ability to easily scale applications on cloud.gov, the site remained up and completely healthy. 
```

I think it's better to keep it concise and leave the fuller quotations for the customer story, except for the one you have below which I agree was the right one to put on the homepage